pkg/lockfile/parse-apk-installed.go

```diff
@@ -0,0 +1,94 @@
+package lockfile
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+const AlpineEcosystem Ecosystem = "Alpine"
+
+func groupApkPackageLines(scanner *bufio.Scanner) [][]string {
+	var groups [][]string
+	var group []string
+
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		if line == "" {
+			if len(group) > 0 {
+				groups = append(groups, group)
+			}
+			group = make([]string, 0)
+			continue
+		}
+		group = append(group, line)
+	}
+
+	if len(group) > 0 {
+		groups = append(groups, group)
+	}
+
+	return groups
+}
+
+func parseApkPackageGroup(group []string, pathToLockfile string) (PackageDetails, error) {
+	var pkg PackageDetails = PackageDetails{}
```

Comment:
Let's set the `Ecosystem` and `CompareAs` here, since we know what those values should be and it reduces the potential for bugs by not having them set conditionally (even though we expect that condition should always be met).

Also, you can omit the type here